extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -150,9 +150,7 @@ export class DeployConfigPage extends DacFxConfigPage {
 			this.model.database = databaseDropdownValue;
 		});
 
-		this.databaseLoader = this.view.modelBuilder.loadingComponent().withItem(this.databaseDropdown).withProperties({
-			required: true
```

Comment:
the required asterisk goes away when this is removed, so required needs to be added to this.databaseDropdown. Same in dacfxConfigPage.ts

Reply:
Good catch - I added it to the FormComponent returned by this function instead since that seems to better match the intended behavior : 

![image](https://user-images.githubusercontent.com/28519865/123160028-8fb65580-d422-11eb-8cb2-0dd7ba6bf3dd.png)

It's an interesting quirk - if the underlying component (in this case the LoadingComponent) has required set it'll use that even if that component doesn't directly support the property.